packages/web-components/fast-foundation/src/checkbox/checkbox.ts

```diff
@@ -115,6 +115,9 @@ export class Checkbox extends FormAssociatedCheckbox {
     constructor() {
         super();
 
+        this.defaultChecked = !!this.checkedAttribute;
```

Comment:
Is this change also related to the TS upgrade?

Reply:
Yes, the transpiled output is functionally equivalent even though TS 4 has stricter checks for `TS2729`:

```
src/checkbox/checkbox.ts:61:45 - error TS2729: Property 'checkedAttribute' is used before its initialization.

61     public defaultChecked: boolean = !!this.checkedAttribute;
                                               ~~~~~~~~~~~~~~~~
```